apps/sms/views/inbox/js/inbox.js

```diff
@@ -55,7 +46,7 @@ var InboxView = {
       'check-uncheck-all-button','composer-link',
       'delete-button', 'edit-header','options-button',
       'settings-button','edit-mode', 'edit-form',
-      'draft-saved-banner'
+      'draft-saved-banner', 'list'
```

Comment:
__question:__ does fast list provide us with something like "item-click" event? So that we can get rid of ```node.dataset...``` logic in ```InboxView.handleEvent``` and work directly with model?